storage/innobase/xtrabackup/test/bootstrap.sh

```diff
@@ -31,6 +31,20 @@ function ssl_version() {
     echo ${sslv}
 }
 
+function glibc_version() {
+    glibc=$(ldd --version | head -1 | awk '{print $NF}')
+    case ${glibc} in
+        2.12|2.17|2.27|2.28|2.31|2.34|2.35) ;;
```

Comment:
how do we know this is valid list of glibc versions?

Reply:
This is the list from which we produce binaries from.